app/forms/sufia/forms/work_form.rb

```diff
@@ -4,6 +4,12 @@ class WorkForm < CurationConcerns::Forms::WorkForm
 
     self.terms += [:collection_ids]
 
+    def initialize(model, current_ability)
+      # Default agreement to accepted if users have already uploaded files to the work
+      @agreement_accepted = model.file_sets.count > 0
```

Comment:
@jcoyne this is where I am setting the value of the `#agreement_accepted` property.